src/main/java/uk/gov/ons/census/casesvc/config/AppConfig.java

```diff
@@ -51,6 +55,25 @@ public MapperFacade mapperFacade() {
     return mapperFactory.getMapperFacade();
   }
 
+  @Bean
+  public PubSubTemplate pubSubTemplate(
+      PublisherFactory publisherFactory,
+      SubscriberFactory subscriberFactory,
+      JacksonPubSubMessageConverter jacksonPubSubMessageConverter) {
+    PubSubTemplate pubSubTemplate = new PubSubTemplate(publisherFactory, subscriberFactory);
+    pubSubTemplate.setMessageConverter(jacksonPubSubMessageConverter);
+    return pubSubTemplate;
+  }
+
+  @Bean
+  public JacksonPubSubMessageConverter jacksonPubSubMessageConverter() {
+    ObjectMapper objectMapper = new ObjectMapper();
+    objectMapper.registerModule(new JavaTimeModule());
+    objectMapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+    objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+    return new JacksonPubSubMessageConverter(objectMapper);
```

Comment:
I've created an ObjectMapperFactory now (https://github.com/ONSdigital/census-rm-case-processor/pull/180) and I want to use that everywhere so there's only one place we have to mess with the Jackson settings